base_layer/core/src/base_node/comms_interface/inbound_handlers.rs

```diff
@@ -157,13 +157,13 @@ where T: BlockchainBackend + 'static
                     .map(|tx| (**tx).clone())
                     .collect();
 
-                let block_template = NewBlockTemplate::from(
+                let mut block_template = NewBlockTemplate::from(
                     BlockBuilder::new()
                         .with_header(header)
                         .with_transactions(transactions)
                         .build(),
                 );
-
+                block_template.body.do_cut_through();
```

Comment:
I feel it would be better to have the cut through call happen in the `build()` of the block builder. That way we can't accidentally forget about it.

Reply:
done